### PR TITLE
Feature/rep 39778 adding http client to mfs

### PR DESCRIPTION
**Pull request by IdoMagor**
Includes the following:

- SDP Parser for fetching the ports of the allocated SDP by RTPEngine himself.
- HTTP module which sends after fetching the ports(using the SDP Parser) and send them to the configured MFS URL.
- Updated RTPEngine call handler which prints logs for getting more info about the call so we could use the logs of SIPREC in our CloudWatch in a better and faster way to investigate errors.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,7 +5,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-config/local.json
+
```
Comment (shmuel7) [inline .gitignore:8]: 2 options:
1. Remove it.
2. Change to use github file instead of S3 bucket file.

File: config/local.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (shmuel7) [inline config/local.json:1]: Remove the creation of local.json from install scripts

**Review by shmuel7 — CHANGES_REQUESTED** [review]
Remove the creation of local.json from install scripts

File: .gitignore (hunk 1 above)
Comment (IdoMagor) [reply .gitignore:8]: Resolved!
Now the installation use the local.json which is included inside the project inside the .tar file

File: config/local.json (hunk 2 above)
Comment (IdoMagor) [reply config/local.json:1]: Resolved as well, same comment as before.

**Review by shmuel7 — APPROVED** [review]

**Review by shmuel7 — APPROVED** [review]

**Pull request merged**